cmd/eksctl/get.go

```diff
@@ -43,6 +51,7 @@ func getClusterCmd() *cobra.Command {
 	fs := cmd.Flags()
 
 	fs.StringVarP(&cfg.ClusterName, "name", "n", "", "EKS cluster name")
+	fs.IntVar(&pageSize, "page-size", DEFAULT_PAGE_SIZE, "The number of results to return in single query")
```

Comment:
perhaps `--max-results` would be better here...

Reply:
Not sure about renaming to `--max-results` as this implies that it will only return results up to this maximum. 

Kubectl uses `--chunk-size`. How about that instead?

Reply:
Ah, my bad... Sure, let's call it `--chunk-size`.